test/e2e/aws_test.go

```diff
@@ -107,7 +107,10 @@ var _ = Describe("functional tests", func() {
 			machineName = cpMachinePrefix + "-2"
 			createAdditionalControlPlaneMachine(namespace, clusterName, machineName, awsMachineName, bootstrapConfigName)
 
-			// TODO: Deploy a MachineDeployment
+			By("Deploying a MachineDeployment")
+			machineDeploymentName := clusterName + "-worker"
+			createMachineDeployment(namespace, clusterName, machineDeploymentName, 3)
+
```

Comment:
Are these changes needed? It looks like the required changes are in the helper methods rather than here. 

This will also need to be reconciled with https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1247 after it is merged.

Reply:
Yes. I should probably wait for #1247